v7.10 Changelog (#6311)
mattermost/docs: source/install/self-managed-changelog.md

Latest Mattermost Releases:

- [Release v7.10 - Feature Release](#release-v7-10-feature-release)
- [Release v7.9 - Feature Release](#release-v7-9-feature-release)
- [Release v7.8 - Extended Support Release](#release-v7-8-extended-support-release)
- [Release v7.1 - Extended Support Release](#release-v7-1-extended-support-release)

## Release v7.10 - [Feature Release](https://docs.mattermost.com/upgrade/release-definitions.html#feature-release)

**Release day: April 14, 2023**

### Important Upgrade Notes

>  - In the next release, v7.11, the following repositories will be merged into one: ``mattermost-server``, ``mattermost-webapp``, ``focalboard`` and ``mattermost-plugin-playbooks``. Developers should read the updated [Developer Guide](https://developers.mattermost.com/contribute/developer-setup/) for details. Playbooks and Boards will be core parts of the product that cannot be disabled.

Reviewer: ```suggestion
 - In the next release, v7.11, the following repositories will be merged into one: ``mattermost-server``, ``mattermost-webapp``, ``focalboard`` and ``mattermost-plugin-playbooks``. Developers should read the updated [Developer Guide](https://developers.mattermost.com/contribute/developer-setup/) for details. **Playbooks and Boards will be core parts of the product that cannot be disabled**.
```

**IMPORTANT:** If you upgrade from a release earlier than v7.9, please read the other [Important Upgrade Notes](/upgrade/important-upgrade-notes.html).

### Highlights

#### Boards
 - 

#### Playbooks
 - 

#### Calls
 - 

### Improvements

#### User Interface (UI)

>  - Added the ability to set a reminder to read a post at a specific time via the “More” menu in posts.

Reviewer 2: ```suggestion
 - Added the ability to set a reminder to read a message at a specific time via the **More** menu in messages.
```

 - Mentions from muted channels are no longer shown or counted on the browser and desktop tabs.

>  - Updated **System Console** descriptions for **Environment > Developer configuration** settings in the **System Console** to clarify that changes require a server restart to take effect.

Reviewer 2: ```suggestion
 - Updated descriptions for **Environment > Developer** settings in the **System Console** to clarify that changes require a server restart to take effect.
```

>  - The custom user status is now shown in the right-hand side and in the **System Console**.

Reviewer 2: Can we clarify where, specifically, a user's custom status displays in "the right-hand side" and in the System Console, please?

>  - Added the ability to handle multiple emails at once when inviting users.

Reviewer 2: ```suggestion
 - Added the ability to invite multiple people at a time by email to a Mattermost instance.
```

 - Added accessibility support to the date picker.

>  - A feedback survey is displayed during a workspace downgrade process from Cloud Professional to Cloud Free.

Reviewer 2: ```suggestion
 - System admins are prompted to complete a feedback survey during a workspace downgrade process from Cloud Professional to Cloud Free.
```

>  - Migrated the post dot menu to a Material UI (MUI) menu.

Reviewer 2: Does this refer to the message (...) **More** option?

Author: @AshishDhama Do you know?

Reviewer 3: @cwarnermm Yes more button, I have added images for more clarity.
<img width="1438" alt="Screenshot 2023-04-04 at 1 28 05 AM" src="https://user-images.githubusercontent.com/16203333/229613970-fbfc2210-30dc-49a4-959e-dbfb35b53bd0.png">
<img width="328" alt="Screenshot 2023-04-04 at 1 28 09 AM" src="https://user-images.githubusercontent.com/16203333/229613976-954f177e-bcac-4a09-a3f3-4797e53a5eba.png">

#### Administration
 - The ``ServiceSettings.PostEditTimeLimit`` config setting no longer affects Plugins, Shared Channels, Integration Actions, or Mattermost Products.

>  - The app server no longer starts if the telemetry ID in the systems table doesn't exist. Although there is no action required by the administrators, it may be good to be aware of this change.

Author: @isacikgoz ?

>  - The app server no longer starts if the telemetry ID in the systems table doesn't exist. Although there is no action required by the administrators, it may be good to be aware of this change.

Reviewer 4: It would be more of a database issue if the ID doesn't exist. Because we already try to create one if one is not set. Administrators should read the error log and take action against it.

>  - Added additional values to the support packet.

Reviewer 2: Do we want to clarify which values were added?

Author: @coltoneshaw ?

>  - Self-hosted instances will now show invoices in **System Console > Billing && Account > Billing History*** for prior self-serve purchases.

Reviewer 2: ```suggestion
 - Self-hosted instances will now show invoices in **System Console > Billing & Account > Billing History*** for prior self-serve purchases.
```

>  - 404 is now returned if an invoice could not be fetched for self-hosted.

Reviewer 2: ```suggestion
 - A 404 error is now returned if an invoice could not be fetched for a self-hosted deployment.
```

Reviewer 2: Are we now showing a 404, or are we no longer showing a 404 in such a case?
